### PR TITLE
feat(Blossom): add theme

### PR DESCRIPTION
**Pull request by Robatortas**
A theme designed with simplicity, the point of this theme is to be able to have something pleasing to look at while searching for a new song, especially in those late night work sessions.

So, the animations are the main attraction here.
They are made for a smooth nice response for the user, they don't complicate the use of the software.

The borders of images are a little rounded, has a lot of gradients and the bottom playback bar is expanded.

The search bar is also a tiny bit changed.

### CODE REVIEW

**Comment by harbassan** [comment]
Are you sure you want to upload the theme here rather than using your own repo with marketplace? Using your own repo would make maintaining your theme much easier for you.

However, if still do want to add it here youll need to update the themes.md with your theme and screenshots.

**Comment by Robatortas** [comment]
Yes I will upload it here, and let me just change the themes.md, thanks!

**Comment by Robatortas** [comment]
Changed THEMES.md!

**Comment by harbassan** [comment]
One last thing, you need to add your theme's info to the main manifest.json file of the repo rather than create your own.

**Comment by Robatortas** [comment]
Deleted the manifest.json inside the Blossom folder and edited the manifest.json on the root.

**Review by Overrevvv — APPROVED** [review]

File: Blossom/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Made for a dark theme look thats pleasing to the eye.
+
+# IMPORTANT
+
+USE SPICETIFY FOR THIS THEME TO WORK!
+
+link to their amazing repo: https://github.com/spicetify/spicetify-cli
+
+And their website: https://spicetify.app/
```
Comment (harbassan) [inline Blossom/README.md:15]: You don't need this since its already in the spicetify-themes repo, so you can remove it.

**Review by harbassan — CHANGES_REQUESTED** [review]

File: Blossom/README.md (hunk 1 above)
Comment (Robatortas) [reply Blossom/README.md]: Removed!

**Comment by harbassan** [comment]
What happened to the README file? I was only talking about the part where your mentioned spicetify, the rest was fine.

**Comment by Robatortas** [comment]
Added README.md again, and deleted the Spicetify part!

**Review by harbassan — APPROVED** [review]

**Pull request merged**